### ISSUE TITLE
TypeError: window.require.config is not a function

### ISSUE THREAD

**Issue by FrancescoBorzi** [ngstack/code-editor#267]
I'm getting the following error when trying to use `code-editor` version `0.4.8`:

```
core.js:3872 ERROR Error: Uncaught (in promise): TypeError: window.require.config is not a function
TypeError: window.require.config is not a function
    at onGotAmdLoader (ngstack-code-editor.js:110)
    at ngstack-code-editor.js:144
    at new ZoneAwarePromise (zone.js:915)
    at CodeEditorService.push../node_modules/@ngstack/code-editor/__ivy_ngcc__/fesm5/ngstack-code-editor.js.CodeEditorService.loadEditor (ngstack-code-editor.js:99)
    at CodeEditorComponent.<anonymous> (ngstack-code-editor.js:777)
    at step (tslib.es6.js:100)
    at Object.next (tslib.es6.js:81)
    at tslib.es6.js:74
    at new ZoneAwarePromise (zone.js:915)
    at __awaiter (tslib.es6.js:70)
    at resolvePromise (zone.js:836)
    at zone.js:750
    at rejected (tslib.es6.js:72)
    at ZoneDelegate.invoke (zone.js:396)
    at Object.onInvoke (core.js:27800)
    at ZoneDelegate.invoke (zone.js:395)
    at Zone.run (zone.js:153)
    at zone.js:894
    at ZoneDelegate.invokeTask (zone.js:431)
    at Object.onInvokeTask (core.js:27788)
```

My `ng version` output:

```
Angular CLI: 9.0.6
Node: 12.14.0
OS: linux x64

Angular: 9.0.6
... animations, cli, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router
Ivy Workspace: Yes

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.900.3
@angular-devkit/build-angular     0.900.6
@angular-devkit/build-optimizer   0.900.6
@angular-devkit/build-webpack     0.900.6
@angular-devkit/core              9.0.3
@angular-devkit/schematics        9.0.6
@ngtools/webpack                  9.0.6
@schematics/angular               9.0.6
@schematics/update                0.900.6
rxjs                              6.5.4
typescript                        3.7.5
webpack                           4.41.2
```

**Comment by DenysVuika** [comment]
the component does not yet support Angular 9. thanks for reporting this issue

**Comment by FrancescoBorzi** [comment]
thanks @DenysVuika for your answer, do you plan to support it in future?

**Comment by DenysVuika** [comment]
Yes, I will try to upgrade the library soon.

**Comment by DenysVuika** [comment]
Released 1.0.0 with Angular 9 support
